### PR TITLE
adding sink spec

### PR DESCRIPTION
**Pull request by jerrypeng**

### CODE REVIEW

**Comment by jerrypeng** [comment]
@sijie @srkukarni please review

File: pulsar-functions/proto/src/main/proto/Function.proto [hunk 1]
```diff
@@ -64,6 +65,17 @@ message SourceSpec {
     map<string,string> topicsToSerDeClassName = 4;
 }
 
+message SinkSpec {
+    string className = 1;
+    // map in json format
+    string configs = 2;
+
+    // configs used only when source feeds into functions
+    SubscriptionType subscriptionType = 3;
```
Comment (srkukarni) [inline pulsar-functions/proto/src/main/proto/Function.proto]: this does not make sense for sink? Also please update the comments above

File: pulsar-functions/proto/src/main/proto/Function.proto [hunk 2]
```diff
@@ -43,15 +43,16 @@ message FunctionDetails {
     string namespace = 2;
     string name = 3;
     string className = 4;
-    string outputSerdeClassName = 5;
-    string output = 6;
+//    string outputSerdeClassName = 5;
```
Comment (srkukarni) [inline pulsar-functions/proto/src/main/proto/Function.proto]: pl remove instead of commenting

**Comment by jerrypeng** [comment]
@srkukarni thanks for the review. I have addressed your comments.

**Pull request merged**